widgets/features/features.php

```diff
@@ -253,6 +253,7 @@ function get_less_variables( $instance ) {
 			}
 		}
 
+		$less_vars['responsive_breakpoint'] = siteorigin_panels_setting('mobile-width') . 'px';
```

Comment:
Okay, so just default to 520 - gotcha. I'll keep in this mind for future edits and I'll submit a commit now.